shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -189,6 +194,18 @@ void FlutterWindowsEngine::SetSwitches(
   project_->SetSwitches(switches);
 }
 
+double ApproxFrameRate() {
+  DWM_TIMING_INFO timing_info;
```

Comment:
I'd suggest zeroing out these structs as C APIs sometimes assume that the caller knows what they are doing with with the fields of the struct up to `cbSize`. So something like `DWM_TIMING_INFO timing_info = {}` ought to be fine and safe.